pkg/build/registry/buildconfiginstantiate/rest.go

```diff
@@ -241,7 +241,7 @@ func (h *binaryInstantiateHandler) handle(r io.Reader) (runtime.Object, error) {
 	if err != nil {
 		return nil, errors.NewInternalError(fmt.Errorf("unable to connect to node, could not retrieve TLS client config: %v", err))
 	}
-	upgrader := spdy.NewRoundTripper(tlsClientConfig)
+	upgrader := spdy.NewRoundTripper(tlsClientConfig, false)
```

Comment:
all?  any! :)

anyway i guess we wouldn't anticipate a redirect on this codepath, but i'm not sure there's a good reason not to follow a redirect if there is one?